mysql/toddl.go

```diff
@@ -32,7 +32,7 @@ import (
 // Spanner. It uses the source schema in conv.SrcSchema, and writes
 // the Spanner schema to conv.SpSchema.
 func schemaToDDL(conv *internal.Conv) error {
-	schemaForeignKeys := make(map[string]bool)
+	spKeyNames := make(map[string]bool)
```

Comment:
spKeyNames is kind of confusing (since we use it for foreign key constraint names and index names) -- I think key is misleading.
And maybe we should emphasize that this is a set of used names (to manage conflicts).

How about naming this usedNames

I think this is worth a comment. Suggest something like:

	// Tracks Spanner names that have been used for foreign key constraints
	// and indexes. We use this to ensure we generate unique names when
	// we mapping from MySQL to Spanner since Spanner requires all foreign
	// key and index names to be distinct (you can't use the same name 
	// for a foreign key constraint and an index).
	usedNames := make(map[string]bool)

Reply:
added comment